### DISCUSSION TITLE
Discord link broken?

### DISCUSSION THREAD

**Discussion by brizzbuzz** [dagger/dagger#5284] [Help]
Hey, seems like I can't get into the discord.  Anyone else having that problem?

**Comment by vito** [comment]
This link (from the homepage) seems to work for me, but it's hard to tell since I'm already in it: https://discord.gg/dagger-io

**Comment by marcosnils** [comment]
Which like are you referring to? Just checked both the dagger.io homepage and the dagger/dagger GH repo link and both seem to work. :thinking:

**Comment by brizzbuzz** [comment]
very strange, I just tried on a different PC and it works fine.  Thanks :)
